lusidtools/cocoon/transaction_type_upload.py

```diff
@@ -58,3 +58,91 @@ def create_transaction_type_configuration(api_factory, alias, movements):
     )
 
     return response
+
+
+def replace_current_transaction_alias(
+    api_factory, updated_alias, movement, properties=None
+):
+
+    """
+    This function overrides the current transaction types alias with a new configuration
+
+    Parameters
+    ----------
+    api_factory: lusid.utilities.ClientApiFactory
+        The LUSID api factory to use.
+    updated_alias: lusid.models.TransactionConfigurationTypeAlias
+        The alias which we want to update.
+    movements: list[lusid.models.TransactionConfigurationMovementDataRequest]
+        The movements to use for transaction type
+    properties:
```

Comment:
What type should the properties be, what are these can you please add a description and types?

Reply:
It should be dict[str, lusid.PerpetualProperty].
That's what the TransactionConfigurationDataRequest model takes. 
Updated.